installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -77,7 +77,7 @@ include::modules/install-sno-generating-the-install-iso-manually.adoc[leveloffse
 .Additional resources
 
 * See xref:../../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#preparing-to-install-sno[Requirements for installing OpenShift on a single node] for more information about installing {product-title} on a single node.
-* See xref:../../post_installation_configuration/enabling-cluster-capabilities.adoc[Enabling cluster capabilities] for more information about enabling cluster capabilities that were disabled prior to installation.
+* See xref:../../installing/overview/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities] for more information about enabling cluster capabilities that were disabled before installation.
```

Comment:
I will leave these as they are for now because changing them would make them inconsistent with the format of the other links in the assembly, and changing them all would be out of scope for this PR